### PR TITLE
Added screenshots for ACNL and edited game.dat!

### PR DESCRIPTION
**Pull request by MelonSpeedruns**

### CODE REVIEW

**Comment by Leo626** [comment]
Should note that testcases are only done with Nightly which doesn't have the fix for invisible characters and buildings yet.

File: games/animal-crossing-new-leaf/game.dat [hunk 1]
```diff
@@ -19,3 +19,12 @@ author = "Flamboyant_Ham"
 cpu = "Intel Core i7-4720HQ"
 gpu = "GeForce GTX 960M"
 os = "Windows 10 14393"
+
```
Comment (TheKoopaKingdom) [inline games/animal-crossing-new-leaf/game.dat]: You need another `[[ testcases ]]` here.

File: games/animal-crossing-new-leaf/game.dat [hunk 2]
```diff
@@ -19,3 +19,12 @@ author = "Flamboyant_Ham"
...
+title = "0004000000086300"
+compatibility = "1"
+date = "2017-11-8"
```
Comment (TheKoopaKingdom) [inline games/animal-crossing-new-leaf/game.dat]: This isn't properly 0 padded, it should be `2017-11-08`

**Comment by TheKoopaKingdom** [comment]
@MelonSpeedruns please address the review comments, or this will be closed.

**Pull request closed without merging**